bin/plotrapthor

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python
```

Comment:
Shouldn't this be `python3`?

Reply:
I changed it to match `bin/rapthor` and the Prefactor scripts, but after researching it a bit I think you're right and it should be `python3`

Reply:
And is `/usr/bin/env python3` OK, or is `/usr/bin/python3` better?

Reply:
The former is the right way, AFAIK.

Reply:
OK, thanks. I updated all the Python scripts to make the shebang usage uniform throughout.